app/controllers/admin/listing_shapes_controller.rb

```diff
@@ -42,7 +102,7 @@ def update
     end
 
     update_result =
-      update_translations(shape, shape_form)
+      update_translations(@current_community, shape_form)
```

Comment:
Shouldn't this take @current_community.id instead of model object? Also, looking at actual method it seems to expect shape as first parameter?